src/graphql/fragments/actions.graphql

```diff
@@ -25,7 +25,58 @@ fragment ColonyAction on ColonyAction {
   createdAt
   newColonyVersion
   isMotion
+  motionData {
+    ...MotionData
+  }
   colony {
     colonyAddress: id
+    nativeToken {
+      nativeTokenDecimals: decimals
+      nativeTokenSymbol: symbol
+      tokenAddress: id
```

Comment:
Do we need to get that here instead of using the colony data from the context?

Reply:
Staking slider can't access context because it gets opened in the objection modal dialog